src/ngraph/frontend/onnx_import/op/squeeze.cpp

```diff
@@ -33,13 +31,21 @@ namespace ngraph
             {
                 NodeVector squeeze(const Node& node)
                 {
-                    auto data = node.get_ng_inputs().at(0);
-                    std::vector<std::int64_t> axes =
-                        node.get_attribute_value<std::vector<std::int64_t>>("axes", {});
-                    std::vector<std::size_t> normalized_axes = ngraph::normalize_axes(
-                        node.get_description(), axes, data->get_shape().size());
-                    auto axes_node = std::make_shared<default_opset::Constant>(
+                    const auto data = node.get_ng_inputs().at(0);
+                    const auto data_rank = data->get_output_partial_shape(0).rank();
+
+                    NGRAPH_CHECK(
```

Comment:
As above, avoid this check if it's only for `normalize_axis`.

Reply:
Can't avoid it until the other PR is merged.